specs/binding-mqtt.spec/src/main/scripts/io/aklivity/zilla/specs/binding/mqtt/schema/mqtt.schema.patch.json

```diff
@@ -136,6 +136,27 @@
                                         "content":
                                         {
                                             "$ref": "#/$defs/validator"
+                                        },
+                                        "user-properties":
+                                        {
+                                            "title": "Session",
+                                            "type": "array",
+                                            "items":
+                                            {
+                                                "additionalProperties": false,
+                                                "properties":
+                                                {
+                                                    "name":
+                                                    {
+                                                        "title": "Name",
+                                                        "type": "string"
+                                                    },
```

Comment:
What are the trade offs between having this as an array with structures that include name and value, vs named pattern properties representing the user property names, and the values directly associated?